invenio_rdm_records/services/schemas/utils.py

```diff
@@ -54,3 +56,23 @@ def dump_empty(schema_or_field):
         return dump_empty(field.nested)
 
     return None
+
+
+class ISODateString(fields.Date):
+    """ISO8601-formatted date string that allows dumping of null values."""
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        """Serialize an ISO8601-formatted date."""
+        try:
+            if value is not None:
+                value = arrow.get(value).date()
+
+            return super()._serialize(
+                value, attr, obj, **kwargs
+            )
+        except ParserError:
+            return missing
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        """Deserialize an ISO8601-formatted date."""
+        return super()._deserialize(value, attr, data, **kwargs).isoformat()
```

Comment:
This is the variation of the marshmallow-utils one... I guess it can't be helped  ╮(￣ω￣;)╭

Reply:
yes, what we expect (or came to expect) is that exact middle ground between `Date` from marshmallow and `ISODateString` from marshmallow-utils
technically, we could also subclass the latter, and save the `_deserialize`...

Reply:
I'm merging and i've opened https://github.com/inveniosoftware/marshmallow-utils/issues/28